docs/source/author/writing_guide.rst

```diff
@@ -0,0 +1,439 @@
+Writing guide
+##############################################################################
+
+The majority of our text content is written in Markdown, and we also developed
+a program called Kordac to allow you to include HTML elements like images and
+videos with simple text tags.
+
+For example, the following text:
+
+.. code-block:: none
+
+  Try using the following blocks:
+
+  ```scratch
+  when flag clicked
+
+  say [Hello] for [2] secs
+  ```
+
+  **Tips**
+
+  - Make sure all your blocks are “snapped” together in a line like a jigsaw puzzle.
+
+will display as the following on the website:
+
+.. image:: ../_static/img/markdown_example_rendered.png
+  :alt: An image showing the above Markdown syntax rendered as HTML
```

Comment:
while this is a great example of the scratch blocks working amazingly (* highfive * team!), they aren't really a great representation of what Kordac actually looks like. Would prefer to see some image/panel/glossary tags - something that has the `{tag-name attribute='hello'}` kind of syntax